scripts/brofish-ping.sh

```diff
@@ -4,6 +4,8 @@
 # In case bro hangs, need to restart it.
 # -----------------------------------------
 
+shopt -s lastpipe
```

Comment:
Not critical.
This `shopt -s lastpipe` might not be necessary if it was ONLY intended for below change of piped commands, since the last command is a "while" block, which handles the variable all inside the same process